src/engine/win/os.cpp

```diff
@@ -1030,11 +1030,12 @@ bool getOpenDirectory(Span<char> output, const char* starting_dir)
 void copyToClipboard(const char* text)
 {
 	if (!OpenClipboard(NULL)) return;
-	int len = stringLength(text);
+	int len = stringLength(text) + 1;
 	HGLOBAL mem_handle = GlobalAlloc(GMEM_MOVEABLE, len * sizeof(char));
 	if (!mem_handle) return;
 
 	char* mem = (char*)GlobalLock(mem_handle);
+	*(mem + len) = '\0';
```

Comment:
this is not necessary, since following copyString does this

Reply:
I just read the source of copyString again and realized it has `*tmp = 0;`. Removed this line 👍